src/datastore/postgres/migrations/1704924533614-offline.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Offline1704924533614 implements MigrationInterface {
+  name = 'Offline1704924533614';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query('ALTER TABLE "offline_event" DROP COLUMN "topicName"');
```

Comment:
Diverts loose the original address information.

We could add it as a property into the message if we really wanted it though